### DISCUSSION TITLE
Export Project list

### DISCUSSION THREAD

**Discussion by pfmp-404** [chrisvel/tududi#245] [Feature requests]
Hi, I think that the ability to export to xls, csv or pdf the list of tasks of the different projects can be interesting. I think that we must be able to set the date of beginning of Projects, task, etc.
Thanks and keep the good work.

**Comment by chrisvel** [comment]
hey @pfmp-404, are you using tududi? Can you describe what would you want to achieve with the export feature?
Something like a printable report with the project name and its tasks underneath would suffice?
